README.md

```diff
@@ -0,0 +1,53 @@
+# cachified-redis-adapter
+
+[![All Contributors](https://img.shields.io/github/all-contributors/mannyv123/cachified-redis-adapter?color=ee8449&style=flat-square)](#contributors)
+
+An adapter meant to be used with
+[@epic-web/cachified](https://github.com/epicweb-dev/cachified)
+
+## Why
+
+[@epic-web/cachified](https://github.com/epicweb-dev/cachified) originally
+contained all adapters as a part of the same package. However, this can cause an
+issue when an adapter is updated and it becomes a breaking change release for
+the entire `@epic-web/cachified` package. This new npm package helps avoid this
+for the `redis` adapter and moves the adapter to its own package.
+
+## Install
+
+```bash
+npm install cachified-redis-adapter
+```
+
+## Usage
+
+**Meant to be used with `@epicweb-dev/cachified`**
+
+```ts
+import {redisCacheAdapter} from 'cachified-redis-json-adapter'
+
+// create an instance of a redis client to pass to the adapter
+// you will need to define this yourself.
+let redisClient = createRedisClient()
+const redisCache = redisCacheAdapter(redisClient)
+
+// usage with cachified
+return cachified({
+  key: `some-cache-key`,
+  // use the cache we defined above
+  cache: redisCache,
+  getFreshValue: async () => {}, // some function to get fresh values
+  // other cachified optoins
+})
```

Comment:
Add a section linking to `cachified-redis-json` if people want to use that instead.

Reply:
added link under **Usage** section of readme